### DISCUSSION TITLE
Login in InfluxDB UI nicht möglich

### DISCUSSION THREAD

**Discussion by deThommy** [solectrus/solectrus#3801] [Q&A] [answered]
Hallo zusammen,

ich möchte InfluxDB (installiert mit SOLECTRUS) nun auch für das dauerhafte Speichern von Messwerten aus Home Assistant verwenden und dachte mir, dass es sinnvoll ist, nicht eine weitere Instanz von InfluxDB zu erstellen, sondern eben in InfluxDB eine neue Organisation zu erstellen.

Ich komme auch auf das UI von InfluxDB, kann mich dort jedoch nicht einloggen, obwohl ich die korrekten Daten aus der .env eingebe.

Falls das Relevanz hat: Solectrus läuft auf einer Synology NAS im Docker. 

Der zugehörige Fehler aus dem Influxdb-log sieht so aus: 
`lvl=error msg="api error encountered" log_id=0tSOy5oW000 handler=session error="unauthorized access"`

Googlen hat mich nicht wirklich weitergebracht.
- Könnt ihr hier helfen?
- Ist meine Idee grundsätzlich überhaupt sinnvoll oder sollte ich lieber eine zweite Instanz aufbauen?!

Danke für eure Hilfe

**Comment by ledermann** [comment, marked as answer]
Seltsam. Hast du wirklich das richtige Username/Password eingegeben? Mach mal das hier:

```
docker compose config influxdb
```

Die relevanten Zeilen sind (bei einer Standardinstallation) diese:

```
DOCKER_INFLUXDB_INIT_USERNAME: admin
DOCKER_INFLUXDB_INIT_PASSWORD: ExAmPl3PA55W0rD
```

Möglich ist, dass du das InfluxDB selbst irgendwann mal geändert hast (über das UI).

Ansonsten: Ich weiß nicht, ob es so eine gute Idee ist, die InfluxDB von SOLECTRUS noch für andere Zwecke zu benutzen. Es sollte möglich sein (sinnvollerweise in separaten Buckets), aber dann besteht eine Verknüpfung zwischen den Systemen, die möglicherweise irgendwann Probleme bereitet. Denkbar sind z.B. Probleme bei zukünftigen Updates. Ich würde das eher nicht machen.

**Reply by deThommy** [reply]
Hallo Georg, danke für die Hinweise. Dann mache ich das separat, nicht, dass ich hier in Zukunft vor unlösbare Probleme gestellt werde. ;-)

Zum eigentlichen Problem: bin jetzt über das Terminal per SSH in die Logs gegangen und habe das Passwort da rauskopiert, und jetzt geht es! 

Vorher habe ich das über das Synology UI mittels Text Editor kopiert. Die Passwörter sehen optisch auch identisch aus (in der .env). Keine Ahnung, ob die dann anders codiert sind? Ist doch eigentlich Klartext?!

**Reply by ledermann** [reply]
Möglicherweise hast du beim Kopieren versehentlich ein Leerzeichen oder sonst etwas mitkopiert. Ansonsten habe ich keine Erklärung dafür.
